docusaurus/docs-internals/editor-controls/multivalue-controls/padding.md

```diff
@@ -4,28 +4,44 @@ toc_max_heading_level: 4
-| `id`            | `string`                                |       -       | The identifier of the key where the padding will save your data                                                                |
+| `id`            | `string`                                |       -       | The identifier of the key where the `padding` will save your data                                                                |
 | `type`          | `string`                                |       -       | Type should be `"padding"` to use this control                                                                                 |
 | `label?`        | `string`                                |       -       | The label displayed on the left side of the control                                                                            |
+| `icon?`         | `string`       |      -       | Icon name that will be rendered on left side of the control's label. View all [icons](../../icons/).                                             |
 | `position?`     | `number`                                |       -       | The position of the control in toolbar                                                                                         |
+| `roles?` | `Array<Role>` | - | Render the control only if the current user's role matches one of the roles in the provided array. <br /> <br /> **`type Role = "admin" \| "viewer" \| "editor" \| "designer" \| "manager"`** | string`** |
 | `devices?`      | `"all"` \|`"desktop"`  \|    `"responsive"`  |   `"all"`    | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
 | `disabled?`     | `boolean`                               |    `false`    | Configure the condition under which the control is disabled or enabled.                                                        |
+| `states?` | `Array<State>` | [`"normal"`] | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination). |
 | `config?.units` | `Array<Unit>`                    | `["px","%"]`  | The units of measurement displayed next to the slider value.<br/> Accepts CSS units (`"px"`, `"%"`) <br/> <br/> `Unit = "px" \| "%"` |
+| `default?` | `Default` | - | The default control value. <br/> <br/> <b>`Default: { paddingType: "grouped"; padding: 0; paddingSuffix: "px"; paddingTop: 0; paddingTopSuffix: "px"; paddingRight: 0; paddingRightSuffix: "px"; paddingBottom: 0; paddingBottomSuffix: "px"; paddingLeft: 0; paddingLeftSuffix: "px";}`</b> <br/> <br/> |
```

Comment:
about `default`

check one more time your keys, because padding `converter` read from another keys + dont't forget also to check your examples of default + examples from return value
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/3965f1f0-c1a8-4474-bc70-6b188644dba3)